circleci/docker-publish

```diff
@@ -49,7 +49,7 @@ push_ecr_image(){
   ECR_URI=$ECR_ACCOUNT_ID.dkr.ecr.$REGION.amazonaws.com/$REPO:$SHORT_SHA
   if image_exists $ECR_URI > /dev/null; then
       echo "ECR image exists, refusing to overwrite"
-      exit 1
+      return
```

Comment:
This is definitely an improvement. Another thing to add is that since publish is its own step in the new circle config, if subsequent steps fail for transient reasons, we can rerun just the parts that fail, which means this will come up way less.

Reply:
Very true.